wp-content/plugins/ig-wp-api-extensions/endpoints/v0/RestApi_ModifiedContent.php

```diff
@@ -230,6 +234,7 @@ protected function build_query_order_clauses() {
 		return ["menu_order ASC", "post_title ASC"];
 	}
 
+	/* Probably unused function. available_languages uses the function in line 6 */
```

Comment:
I agree. Let's remove the function. Unless @maxammann you had some other intentions here?

Reply:
You can remove them. They don't seem to be used. Those were my functions and you changed them, right?

Reply:
that is correct.